components/EditVocabForm/EditVocabForm.js

```diff
@@ -0,0 +1,152 @@
+import { languages } from "../../lib/languages";
+import styled from "styled-components";
+import useLocalStorageState from "use-local-storage-state";
+import { nanoid } from "nanoid";
+import { StyledForm } from "../StyledForm";
+
+export default function EditVocabForm({
+  word,
+  editId,
+  setEditing,
+  onSaveEdited,
+  setPopup,
+}) {
```

Comment:
I'm not sure what `setEditing` and `setPopup` are supposed to do exactly but you should follow the convention to prefix the prop names with `on...` when you pass callbacks to the component. The idea is that something happens inside your component and this triggers the `onSomethingHappened` event. Other components that use this component can then pass a callback to handle that event.